axlearn/audio/asr_decoder.py

```diff
@@ -28,6 +28,40 @@
 from axlearn.common.utils import Nested, Tensor
 
 
+def is_valid_ctc_seq(logitpaddings, labels, labelpaddings):
```

Comment:
```suggestion
def _is_valid_ctc_seq(*, paddings: Tensor, target_labels: Tensor, target_paddings: Tensor) -> Tensor:
```

In general, can we add type+return annotations and follow existing naming conventions (following https://github.com/apple/axlearn/blob/main/CONTRIBUTING.md) here and elsewhere?

Reply:
Thanks, Done.